solidity/contracts/token/bridge/EverclearTokenBridge.sol

```diff
@@ -429,16 +440,17 @@ contract EverclearEthBridge is EverclearBridge {
     using Address for address payable;
     using TypeCasts for bytes32;
 
+    uint256 private constant SCALE = 1;
+
     /**
      * @notice Constructor to initialize the Everclear ETH bridge
      * @param _everclearAdapter The address of the Everclear adapter contract
      */
     constructor(
         IWETH _weth,
-        uint256 _scale,
         address _mailbox,
         IEverclearAdapter _everclearAdapter
-    ) EverclearBridge(_everclearAdapter, IERC20(_weth), _scale, _mailbox) {}
+    ) EverclearBridge(_everclearAdapter, IERC20(_weth), SCALE, _mailbox) {}
```

Comment:
Do we plane to use `SCALE` elsewhere? Maybe we can add `DEFAULT_SCALE` to `TokenRouter`.